src/MSBuild.Conversion.Project/ProjectRootElementExtensionsForConversion.cs

```diff
@@ -528,5 +548,22 @@ public static IProjectRootElement RemoveWebExtensions(this IProjectRootElement p
 
             return projectRootElement;
         }
+
+        public static IProjectRootElement RemoveXamarinImport(this IProjectRootElement projectRootElement, ProjectStyle projectStyle)
```

Comment:
This may not be necessary. I think you can move this logic to be under `ChangeImportsAndAddSdkAttribute`. I guess that could also get factored out into two methods as well, one for removing things and another for adding the SDK attribute. But I think it'd be better to just make the change there for now.

Reply:
Sounds good! I'll move it inside `ChangeImportsAndAddSdkAttribute`